### PR TITLE
HristovY

### PR DESCRIPTION
**Pull request by anaseya**

### CODE REVIEW

File: HristovY/JavaScript-code.js [hunk 1]
```diff
@@ -0,0 +1,142 @@
...
+ * @link http://www.ecma-international.org/ecma-262/7.0/index.html
+ * @link https://developer.mozilla.org/en/docs/Web/JavaScript
+ * @link http://contribute.jquery.org/style-guide/js/
+ * @link http://www.w3schools.com/js/js_conventions.asp
+ * @link https://google.github.io/styleguide/javascriptguide.xml
+ */
+
+var addScroll = false; // @type {boolean} addScroll
+
+if ((navigator.userAgent.indexOf('MSIE 5') > 0) || (navigator.userAgent.indexOf('MSIE 6')) > 0) {
```
Comment (victorlazov) [inline HristovY/JavaScript-code.js:15]: Line width == 96 chars.

File: HristovY/JavaScript-code.js [hunk 2]
```diff
@@ -0,0 +1,142 @@
...
+}
+
+var b = navigator.appName; // @type {string} b - Represents the name of the browser.
+var pX = 0;                // @type {number} [pX=0]
+var pY = 0;                // @type {number} [pY=0]
+document.onmousemove = mouseMove;
+
+if (b == "Netscape") {
+    document.captureEvents(Event.MOUSEMOVE)
+};
```
Comment (victorlazov) [inline HristovY/JavaScript-code.js:26]: I think this is a type, but nevertheless the semicolon is on the wrong row.

File: HristovY/JavaScript-code.js [hunk 3]
```diff
@@ -0,0 +1,142 @@
...
+
+        if ((pX + 120) > window.innerWidth) {
+            pX = window.innerWidth - 150;
+        }
+
+        theLayer.left = pX + 10;
+        theLayer.top = pY + 15;
+        theLayer.visibility = 'show';
+    }
+	else {
```
Comment (victorlazov) [inline HristovY/JavaScript-code.js:72]: Wrong indent.

File: HristovY/JavaScript-code.js [hunk 4]
```diff
@@ -0,0 +1,142 @@
...
+function ShowMenu2() {
+        if (b == "Netscape") {
+            theLayer = eval('document.layers[\'menu2\']');
+            theLayer.visibility = 'show';
+        } 
+		else {
+            theLayer = eval('document.all[\'menu2\']');
+            theLayer.style.visibility = 'visible';
+        }
+}
```
Comment (victorlazov) [inline HristovY/JavaScript-code.js:142]: WTF is this?

File: HristovY/JavaScript-code.js [hunk 5]
```diff
@@ -0,0 +1,142 @@
...
+    document.captureEvents(Event.MOUSEMOVE)
+};
+
+/**
+ * [Short function description here - if needed!].
+ * @param {object} evn - [Short parameter description here].
+ * @return void
+ */
+function mouseMove(evn) {
+	if (b == "Netscape") {
```
Comment (akraychev) [inline HristovY/JavaScript-code.js:34]: Indent should be the same for if and else

File: HristovY/JavaScript-code.js [hunk 6]
```diff
@@ -0,0 +1,142 @@
...
+    if (b == "Netscape") {
+        document.layers['menu2'].visibility = 'hide';
+    } 
+	else {
+        document.all['menu2'].style.visibility = 'hidden';
+    }
+}
+
+function ShowMenu2() {
+        if (b == "Netscape") {
```
Comment (akraychev) [inline HristovY/JavaScript-code.js:134]: indent in different function should be the same

**Pull request merged**